package.json

```diff
@@ -30,14 +30,14 @@
     "external-services": "docker compose --file packages/hash/external-services/docker-compose.yml --env-file .env",
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
     "@todo.1": "Remove the ignore argument below for mock-block-dock once repo is upgraded fully to 0.2 spec",
-    "fix:dependency-version-consistency": "check-dependency-version-consistency --fix . --ignore-dep=mock-block-dock --ignore-dep=@blockprotocol/graph --ignore-dep=@blockprotocol/hook --ignore-dep=block-scripts",
+    "fix:dependency-version-consistency": "check-dependency-version-consistency --fix . --ignore-dep=mock-block-dock --ignore-dep=@blockprotocol/hook --ignore-dep=@blockprotocol/graph --ignore-dep=@blockprotocol/hook --ignore-dep=block-scripts",
```

Comment:
as Ciaran mentioned in the same file, `hook` also appears twice here

Reply:
Now redundant